description: debian specific instructions when no working sandbox is available
author: Michael Gilbert <mgilbert@debian.org>

--- a/content/browser/zygote_host/zygote_host_impl_linux.cc
+++ b/content/browser/zygote_host/zygote_host_impl_linux.cc
@@ -114,14 +114,13 @@ void ZygoteHostImpl::Init(const base::Co
     // root.
     use_suid_sandbox_for_adj_oom_score_ = use_suid_sandbox_;
   } else {
-    LOG(FATAL)
-        << "No usable sandbox! Update your kernel or see "
-           "https://chromium.googlesource.com/chromium/src/+/master/"
-           "docs/linux/suid_sandbox_development.md for more information on "
-           "developing with the SUID sandbox. "
+    LOG(ERROR)
+        << "No usable sandbox! If this is a Debian system, please install the "
+           "chromium-sandbox package to solve this problem. "
            "If you want to live dangerously and need an immediate workaround, "
            "you can try using --"
         << sandbox::policy::switches::kNoSandbox << ".";
+    exit(EXIT_FAILURE);
   }
 }
 
